2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 27 Line 2_
```markdown
```python
date_index = 105
```
Would be really nice to have a concluding section, where the results for each approach are compared, and maybe give some pros and cons of each (in that case maybe no need to mention them in detail in each section, as I suggested above).

_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._

Reply:
This is part of the presentation